mteb/overview.py

```diff
@@ -133,6 +133,18 @@ def filter_tasks_by_modalities(
         return [t for t in tasks if _modalities.intersection(t.modalities)]
 
 
+def filter_aggregate_tasks(
+    tasks: list[AbsTask], exclude_aggregate: bool = True
+) -> list[AbsTask]:
+    """Args:
+    tasks: A list of tasks to filter.
+    exclude_aggregate: Is True, filter out aggregate tasks. If False, return all the original tasks.
+    """
+    if exclude_aggregate:
+        return [t for t in tasks if not t.is_aggregate]
+    return tasks
```

Comment:
The `exclude_aggregate` flag has to be True before this function is called. Let's omit the same check and directly return the filtered list.

I see that filter_tasks_by_modalities should be similarly updated as well. It is separate and I will raise a new issue for that.

```suggestion
    return [t for t in tasks if not t.is_aggregate]
```